### PR TITLE
Add -S/--SHA256 option and make SHA384 default

### PR DESCRIPTION
**Pull request by asgibbons**

### CODE REVIEW

File: python/pacsign/pacsign/pacsign.py [hunk 1]
```diff
@@ -200,8 +206,8 @@ def main():
             False, "Invalid key manager module %s" % args.HSM_manager
         )
 
-    if args.cert_type == "RK_384":
-        common_util.assert_in_error(False, "384-bit keys not supported")
+    #if args.cert_type == "RK_384":
+    #    common_util.assert_in_error(False, "384-bit keys not supported")
```
Comment (fpgamatt) [inline python/pacsign/pacsign/pacsign.py:210]: I would just go ahead and delete these two lines.

Comment (asgibbons) [reply python/pacsign/pacsign/pacsign.py]: Done.

**Comment by dmlomartire** [comment]
I see the second user image (fim2) was added as a new image type.  What about factory image?  That was also a new image type that they want to remotely update.  Will that be handled via a separate pull request?

**Comment by fpgamatt** [comment]
I don't think Scott was aware of a second factory image at the time of this PR.

**Review by fpgamatt — APPROVED** [review]

**Comment by dmlomartire** [comment]
> I don't think Scott was aware of a second factory image at the time of this PR.

Actually, the request is the initial factory image (not a second).  Right now, there is no way to remote update the factory image (BBS).  That was a new request for Arrow Creek.

**Pull request merged**